packages/framework/src/Component/Elasticsearch/ElasticsearchStructureManager.php

```diff
@@ -152,4 +266,14 @@ protected function getDefinition(int $domainId, string $index): array
 
         return $definition;
     }
+
+    /**
+     * @param int $domainId
+     * @param string $index
+     * @return array
+     */
+    public function getStructureDefinition(int $domainId, string $index): array
+    {
+        return $this->getDefinition($domainId, $index);
```

Comment:
Instead of calling the deprecated method, you should move the code here and call `$this->getStructureDefinition($domainId, $index);` from the deprecated method.